src/ui_ng/lib/src/confirmation-dialog/confirmation-dialog.component.ts

```diff
@@ -51,7 +55,29 @@ export class ConfirmationDialogComponent {
         this.opened = true;
     }
 
+    get batchOverStatus(): boolean {
+        if (this.batchInfors.length) {
```

Comment:
Use explicit statement to avoid confusion next time. if (this.batchInfors.length>)